packages/api/client.ts

```diff
@@ -367,6 +371,16 @@ export const getServiceStops = async (dbClient: Kysely<Database>, input: Service
         return [];
     }
 
+    const tracks = await dbClient
```

Comment:
Could we do this before the datasource query?

Reply:
done